Alex-App/src/app/list/list.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from "@angular/core";
+import { MyServiceService } from "../my-service.service";
+import { Member } from "../models/member.model";
+
+@Component({
+  selector: "app-list",
+  templateUrl: "./list.component.html",
+  styleUrls: ["./list.component.css"]
+})
+export class ListComponent implements OnInit {
+  members: Member[];
+
+  constructor(private myService: MyServiceService) {}
+
+  ngOnInit() {
+    this.fetchMembers();
+  }
+
+  async fetchMembers(): Promise<Member | void> {
+    try {
+      const getMembers = await this.myService.getMembers();
+
+      getMembers.data.sort((a, b) =>
```

Comment:
If i read the requirements right it seems too. The list that is displayed on the page is sorted by the emailAdress property from the data that is returned. Totally possible that I misunderstood what you wanted though. The image attached shows what is originally returned in the console & what the .sort() does in the actual component 
![list](https://user-images.githubusercontent.com/42323618/65781444-a29ad980-e119-11e9-8f55-f248bbc3450e.png)

Reply:
If it works it works. I am just surprised since the documentation says the array wouldn't be altered. Normally its like `const sortedArray = unsortedArray.sort();`